.github/workflows/release.yml

```diff
@@ -10,25 +10,27 @@ jobs:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
     steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-      with:
-        # Ensure that git uses your token with admin access to the repo
-        token: ${{ secrets.ADMIN_TOKEN }}
-        fetch-depth: 0
-    - run: git fetch --tags --depth=1 origin +refs/tags/*:refs/tags/*
-    - name: Setup node
-      uses: actions/setup-node@v4
-      with:
-        node-version: 18
-        cache: 'npm'
-        registry-url: 'https://registry.npmjs.org'
-    - name: Auto release
-      env:
-        GH_TOKEN: ${{ secrets.GH_TOKEN }}
-        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-        SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
-      run: |
-        npm ci
-        npx auto shipit
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Ensure that git uses your token with admin access to the repo
+          token: ${{ secrets.ADMIN_TOKEN }}
+          fetch-depth: 0
+      - run: git fetch --tags --depth=1 origin +refs/tags/*:refs/tags/*
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: 'npm'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Auto release
+        env:
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
+        run: |
+          npm ci
+          cd packages/ui
+          npm run build
+          npx auto shipit
```

Comment:
I'm _pretty_ sure this executes for the full repo, but worth double checking docs to confirm it finds the root or just cd-ing back up to root 🤷🏼‍♂️